extensions/binding/org.eclipse.smarthome.binding.tradfri/ESH-INF/thing/thing-types.xml

```diff
@@ -71,6 +71,28 @@
         </config-description>
     </thing-type>
 
+    <thing-type id="0200">
```

Comment:
In case of a color lamp with adjustable color temperature and brightness the corresponding ZigBee ID should be 210 instead of 200.

Reply:
The _real_ device ID of this lamps is 200 (0x0200), please see #4251 (confirmed on hardware).
Do we take the _real_ device ID, or the device ID that we imagine that is correct?
The whole thingType naming based on technical device IDs, that the IKEA gateway do not even expose, is anyway a bit.. hmm

Reply:
When creating it this way I was hoping that IKEA would stay close to the specification, which would be the best way to figure out the capabilities of the device. This is what we ended up with in the hue binding after many 3rd party devices were supported by it. I'd guess that sooner or later IKEA will come to the same conclusion and our approach will prove to be the right choice ☺️

Reply:
Sry didn't look into #4251. You already complained your choice. And this is another break with the ZigBee protocol caused by IKEA developers.

To be compliant I would suggest to use the same thing-type definition as we use in the Hue and other bindings. In this case it should be the device ID of what we think the bulb is capable of -> 210. If not, we should remove the color_temperature channel as well (where later one is IMHO the worse way).

btw. I didn't mentioned it before: You did a great job so far.

Reply:
My personal feeling is also that 210 is the right choice, and not 200. But as the _real_ device ID was 210, I renamed it so stick with the previous naming scheme.
I'm happy to re-rename it to 210, cause this better reflects the capabilities of the Thing.

P.S: After re-reading my previous comment, it sounded a bit offensive. That was not intentional.

Reply:
> But as the real device ID was 210, ...

You mean 200, I guess. ;-) Okay, we give it a try.

> ... it sounded a bit offensive.

Oh no, everything was alright. I am fine with it.

Reply:
Yes, 200... All these numbers!
I changed it now to *210* in the binding and the documentation. PR update ongoing with the other changes.

Reply:
Hm, my comment above actually meant that we SHOULD go for the real Zigbee type, because sooner or later other devices will appear with the 210 type - and maybe they then need a different logic to be supported (e.g. hardware instead of software color temp setting). So I was actually fine with the "200" choice.

>  If not, we should remove the color_temperature channel as well

@cweitkamp The other option would be that we think about doing "software calculation" on those (hue) as well - do you think that would be possible?

Reply:
But I don't have a too strong opinion on 200 vs. 210 - if you both now agreed that 210 is the better choice, so be it. We can still revisit this once we see more devices that bring us more clarity on how the gateway handles them.

Reply:
Well, @cweitkamp has also good arguments.
I'm absolutely against removing the color_temperature channel from the new thing. See my discussion above; but replace "brightness" with "color temperature". And this can not be handled with the color channel, at least not without rules or something.